MODULE.bazel

```diff
@@ -1,8 +1,11 @@
 "Bazel dependencies"
 
+# replaced by export-subst during 'git archive'
+VERSION = "$Format:%(describe:tags=true)$"
```

Comment:
What does this evaluate to at a commit that doesn't have a tag? I frequently depend on specific unreleased commits and ideally we would produce a valid version in that case.

Reply:
Only `git archive` does any replacement. So "depend on the ruleset at specific commit" means you'll see exactly this string, nothing evaluates it.

Reply:
@alexeagle Are you sure that's true for GitHub's `/archive/<sha>.zip` endpoint? I thought that that uses `git archive`.